test/tests_linux_ie_cpu.txt

```diff
@@ -17,14 +17,18 @@ tests_common.txt
 [RUN]
 # Specify tests/patterns/regex that should be included
 
-
-
 ###################################################
 [SKIP]
 # Specify tests/patterns/regex that should be excluded/skipped
 
 # [PARAMETER_MISMATCH] Failed to set Blob with precision not corresponding to user output precision
 NGraphExecTest.Axpy8bit
+MathOps.AnyKeepDims
+MathOps.AnyNegativeAxis
+MathOps.AnyPositiveAxis
+MathOps.AllKeepDims
+MathOps.AllNegativeAxis
+MathOps.AllPositiveAxis
```

Comment:
Why do these get added? Are the failures indirectly related to FloorMod?

Reply:
No, these tests were individually disabled and not being run on IE or CPU or Interpreter. I moved them to the manifest after enabling and checking the status.